gef.py

```diff
@@ -10644,26 +10661,21 @@ def parse_monitor_info_mem() -> Generator[Section, None, None]:
                           offset=off,
                           permission=perm)
 
-    @staticmethod
-    def parse_info_mem():
```

Comment:
Needed by BMP

Reply:
That's exactly what I meant in the comment above: since this case only applies to bmp alone it would make perfect sense for bmp to inherit the base memory manager and use its way for parsing memory, and then expose to gef.
And if that way (or any other) become "mainstream" enough then we can include it in gef itself.

We can't (and shouldn't) handle every single custom case everyone can be facing,  but we can provide a way to easily plug their own to gef and benefit from everything else.

That is the whole intent behind writing all those "managers": help make gef smaller but more flexible to handle any case

Reply:
Sure we could, but we'd need to copy it for both BMP and openocd, so that would preclude people from just importing the single file, and we drift even farther from the one-file solution. Plus, if the syntax is determined by GDB, then to me it makes sense to offer it as an implementation for extras to use.

Even if we don't, we can still have other memory managers or architectures that need to use the existing ones, which you're hiding from them.